### PR TITLE
fix(lineage): fixed ui crash and hanging load for lineage display

### PR DESCRIPTION
**Pull request by ciyer**
For certain files, there seems to be problems with the request for a notebook server and the request for lineage that causes the UI to either hang when displaying lineage or crash entirely. 

Compare 

https://dev.renku.ch/projects/cramakri/renku-tutorial-flights/files/lineage/notebooks/00-FilterFlights.ipynb

With 

https://sekhar.dev.renku.ch/projects/cramakri/renku-tutorial-flights/files/lineage/notebooks/00-FilterFlights.ipynb

### CODE REVIEW

File: src/api-client/project.js [hunk 1]
```diff
@@ -354,7 +354,7 @@ function addProjectMethods(client) {
     const headers = client.getBasicHeaders();
     const url = `${client.baseUrl}/notebooks/${projectPath}/${commitSha}`;
     return client.clientFetch(url, { method: 'GET', headers})
-      .then(resp => resp.data)
+      .then(resp => resp.data, resp => ({url: null}))
       .then(server => server.url ? server.url : null)
```
Comment (vfried) [inline src/api-client/project.js]: Do we need the two lines of code here? (I ask because i'm not sure)

Comment (vfried) [reply src/api-client/project.js]: I mean the 357 and 358

Comment (ciyer) [reply src/api-client/project.js]: You are correct, data returns data, not a promise, so there is no need to wait on it. I think this code structure is a holdover from an earlier implementation where there was an extra promise there.

Comment (vfried) [reply src/api-client/project.js]: ok, we should link this PR to the one that lorenzo has in progress, so he knows about this changes.

**Pull request merged**